### ISSUE TITLE
Unable to parse empty arrays with prettier 3.3.0

### ISSUE THREAD

**Issue by sepek** [electrovir/prettier-plugin-multiline-arrays#32]
After upgrading prettier `3.3.0` to I get for the following error:
```
Error: Found "[" but: its sibling was not an indent Doc.: 
[error]     at […]/node_modules/prettier-plugin-multiline-arrays/dist/printer/insert-new-lines.js:57:23
[error]     at walkDoc ([…]/node_modules/prettier-plugin-multiline-arrays/dist/printer/child-docs.js:28:10)
[error]     at […]/node_modules/prettier-plugin-multiline-arrays/dist/printer/child-docs.js:41:20
[error]     at Array.every (<anonymous>)
[error]     at walkDoc ([…]/node_modules/prettier-plugin-multiline-arrays/dist/printer/child-docs.js:40:18)
[error]     at […]/node_modules/prettier-plugin-multiline-arrays/dist/printer/child-docs.js:41:20
[error]     at Array.every (<anonymous>)
[error]     at walkDoc ([…]/node_modules/prettier-plugin-multiline-arrays/dist/printer/child-docs.js:40:18)
[error]     at insertLinesIntoArray ([…]/node_modules/prettier-plugin-multiline-arrays/dist/printer/insert-new-lines.js:17:30)
[error]     at printWithMultilineArrays ([…]/node_modules/prettier-plugin-multiline-arrays/dist/printer/insert-new-lines.js:419:15)
```

Example code which triggers it:
```
let errorMessages = [];
```

Code which works around it:
```
let errorMessages = [
  null,
];
```

Works fine without using `prettier-plugin-multiline-arrays` (except I won't have multiline arrays anymore).

**Comment by SiNONiMiTY** [comment]
Confirming this issue on empty arrays. Staying on prettier@3.2.0 for the meantime

**Comment by electrovir** [comment]
The fix from @richardsimko has been published to npm in [v3.0.6](https://www.npmjs.com/package/prettier-plugin-multiline-arrays/v/3.0.6). Thank you so much @richardsimko for the PR! 💙
